src/search/airportsearch.cpp

```diff
@@ -403,7 +403,6 @@ void AirportSearch::saveState()
   atools::gui::WidgetState widgetState(lnm::SEARCHTAB_AIRPORT_WIDGET);
   widgetState.save(airportSearchWidgets);
   saveViewState(viewStateDistSearch);
-  // TODO: make sure RFG state (min, max, predefined) is saved and restored.
 }
```

Comment:
restore does not yet work for me, I haven't found the reason. Perhaps the min/max setting ?

Reply:
I already fixed the restore with c9b649caee99a8e3b0b3d5704fb6548f31703d90
Alex

Reply:
yes, i saw that. I think i have pulled that and built lnm with that. My built version still does not restore the values.

Reply:
Will check.